client/admin_test.go

```diff
@@ -129,6 +129,13 @@ func Example_permission() {
 	context := testutils.NewRootTestBaseContext()
 	client := client.NewAdminClient(context, s.ServingAddr(), client.Permission)
 
+	// The test server creates a permission config entry for 'server.TestUser'.
+	// Delete it first so it does not interfere with our configs.
+	err := client.Delete(server.TestUser)
```

Comment:
it can't be that flexible since we need pre-generated certs for it. If we want custom entries (useful if testing in insecure mode), TestServer.WritePermissionConfig would do the trick.